Sources/Unbox.swift

```diff
@@ -482,6 +482,20 @@ public class Unboxer {
     
     /// Unbox a required Array containing values of a raw type
     public func unbox<T: UnboxableRawType>(key: String, isKeyPath: Bool = true) -> [T] {
+        if let rawValue = UnboxValueResolver<[T]>(self).resolveOptionalValueForKey(key, isKeyPath: isKeyPath) {
```

Comment:
Maybe name it `rawValueArray` or `rawValues` to indicate that it's an array of values, rather than a single value?